.github/workflows/codeql.yml

```diff
@@ -12,4 +12,7 @@ on:
 
 jobs:
   analyze-java:
-    uses: coveo/actions/.github/workflows/java-maven-openjdk11-codeql.yml@main
+    uses: coveo/public-actions/.github/workflows/java-maven-openjdk11-codeql.yml@main
+    with:
+      runs-on: "['linux', 'x64', 'ec2.instance-type = t3.large']"
+      mvn-arguments: ${{ inputs.mvn-arguments }}
```

Comment:
I don't see any inputs in the workflow, you're sure of this?

Reply:
Yes... I think :P 
https://github.com/coveo/public-actions/blob/main/.github/workflows/java-maven-openjdk11-codeql.yml#L6